storage/client/types.go

```diff
@@ -3,8 +3,11 @@ package client
 
 import (
 	api "github.com/oasisprotocol/nexus/api/v1/types"
+	"github.com/oasisprotocol/nexus/common"
 )
 
+type BigInt = common.BigInt
```

Comment:
this was the only relevant part from the now-deleted `storage/client/api.go`, which belongs here anyway